README.md

```diff
@@ -116,6 +116,16 @@ Everything you need is available prebuild in FreeBSD' ports.
 
     pkg install gmake node
 
+#### Arch:
+
+You can install cjdns by running
+
+	pacman -S cjdns
```

Comment:
This line's indented with a tab, while the rest of them are indented with spaces.  You might want to turn on visible tabs in your editor.